tcf_website/views/index.py

```diff
@@ -1,19 +1,22 @@
 """Views for index and about pages."""
 import json
-from django.shortcuts import render, redirect
+from django.shortcuts import render
 from django.views.generic.base import TemplateView
 
 
 def index(request):
     """
     Index view.
 
-    Redirect to landing page if user not authorized, otherwise show
-    browse page.
+    Redirect to landing page if user not authorized.
     """
-    if request.user.is_authenticated:
-        return redirect('browse')
-    return render(request, 'landing/landing.html')
+
+    # Load "About Team" data from json file
+    with open('tcf_website/views/team_info.json') as data_file:
+        team_info = json.loads(data_file.read())
```

Comment:
You can probably replace this with `team_info = json.load(data_file)`
https://stackoverflow.com/questions/20199126/reading-json-from-a-file